src/templates/app/_includes/authenticate_message.jsx

```diff
@@ -45,7 +45,7 @@ const FileSelector = ({
                                 const j = i + 1;
                                 return (
                                     <React.Fragment key={i}>
-                                        <h3>{document.name}</h3>
+                                        <h3 data-show-onfido={(type === 'nigeria' && !document.is_nimc) ? document.value : ''}>{document.name}</h3>
```

Comment:
we can combine the logic from authenticate.js instead of adding ternary in template

Reply:
We have a selfie section for Nigeria, and there are many changes for poi part in the DOM as well. So IMO if we want to handle this completely from `authenticate.js`, we need to inject or change DOM using JS. This will add complexity to the code.